spectree/plugins/flask_plugin.py

```diff
@@ -194,7 +194,8 @@ def validate(
         before(request, response, req_validation_error, None)
         if req_validation_error:
             after(request, response, req_validation_error, None)
-            abort(response)  # type: ignore
+            assert response  # make mypy happy
+            abort(response)
```

Comment:
This was weird. Removing `type: ignore` here passed when running it on my machine, but it was raising error when running here. When I put it back, it was complaining about unused ignore. So the way how to oveercome it was to make sure the response is not `Optional`.

Reply:
Sometimes that happens even if the Python and mypy versions are the same. I'm not sure about the root cause.